pkgs/fixnum/README.md

```diff
@@ -0,0 +1,14 @@
+[![Dart CI](https://github.com/dart-lang/core/actions/workflows/fixnum.yaml/badge.svg)](https://github.com/dart-lang/core/actions/workflows/fixnum.yaml)
+[![Pub](https://img.shields.io/pub/v/fixnum.svg)](https://pub.dev/packages/fixnum)
+[![package publisher](https://img.shields.io/pub/publisher/fixnum.svg)](https://pub.dev/packages/fixnum/publisher)
+
+A fixed-width 32- and 64- bit integer library for Dart.
+
+Provides data types for signed 32- and 64-bit integers.
+The integer implementations in this library are designed to work identically
+whether executed on the Dart VM or compiled to JavaScript.
+
+## Publishing automation
```

Comment:
Let's also delete this text from the readme - `Publishing automation` and related. All that info can now live at the repo root.